backend/src/backend/app.py

```diff
@@ -114,6 +128,39 @@ async def get_tasks(user_id: Annotated[str, Depends(get_user_id)]):
     del user_id
     return [{"id": 1, "title": "Buy Milk"}, {"id": 2, "title": "Buy Bread"}]
 
+# Open AI chat
+openai.api_key = OPENAI_API_KEY 
+
+# Pydantic models for request and response
+class ChatRequest(BaseModel):
+    message: str
+
+class ChatResponse(BaseModel):
+    response: str
+
+@app.post("/api/chat", response_model=ChatResponse, status_code=status.HTTP_200_OK)
+async def chat(chat_request: ChatRequest):
+    user_message = chat_request.message
+
+    if not user_message:
```

Comment:
I think this validation is already done on the FastAPI side, and we don't need to write it.
Writing tests might provide how this API endpoint works. Having tests also helps reviewers understand the logic by providing input/output examples.

Please feel free to reach out to me if you need some help.